src/Raven.Server/Config/Categories/MemoryConfiguration.cs

```diff
@@ -71,5 +73,11 @@ public MemoryConfiguration()
         [TimeUnit(TimeUnit.Seconds)]
         [ConfigurationEntry("Memory.TemporaryDirtyMemoryChecksPeriodInSec", ConfigurationEntryScope.ServerWideOnly)]
         public TimeSetting TemporaryDirtyMemoryChecksPeriod { get; set; }
+        
+        [Description("The minimum swap size. If the swap size is lower a notification will arise")]
+        [DefaultValue(DefaultValueSetInConstructor)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [ConfigurationEntry("Memory.MinSwapSize", ConfigurationEntryScope.ServerWideOnly)]
+        public Size MinSwapSize { get; set; }
```

Comment:
- let's put it in `PerformanceHintsConfiguration.cs` file
- configuration key - `"PerformanceHints.Memory.MinSwapSizeInMb"`